CONTRIBUTING.md

```diff
@@ -0,0 +1,36 @@
+# Contributors Guide
+
+Thank you for considering contributing to GeophysicalFlows.jl! 
+
+Please feel free to ask us questions and chat with us at any time if you're
+unsure about anything.
+
+[issue](https://github.com/FourierFlows/GeophysicalFlows.jl/issues) or starting a [discussion](https://github.com/FourierFlows/GeophysicalFlows.jl/discussions).
+
+Best way to get in touch is to either just open a GitHub [issue](https://github.com/FourierFlows/GeophysicalFlows.jl/issues) 
```

Comment:
```suggestion
Best way to get in touch is to either just open a GitHub [issue](https://github.com/FourierFlows/GeophysicalFlows.jl/issues) 
```
This seems to be a leftover of re-arranging the sentence. Otherwise, this looks great to me :+1: